tracer/test/snapshots/Security.AspNetCore2.__test=blocking_enableSecurity=True_expectedStatusCode=403_url=_.verified.txt

```diff
@@ -17,8 +17,6 @@
       http.request.headers.host: localhost:00000,
       http.request.headers.user-agent: Mistake Not... Hello/V,
       http.request.headers.x-forwarded: 86.242.244.246,
-      http.response.headers.content-length: 2xxx,
-      http.response.headers.content-type: text/html,
```

Comment:
Are these changes correct? 🤔 Aren't we expecting these changes to not change the effect of blocking, just how we do it?

Reply:
Yes, we're writing the response happens after we write the tag headers so these headers aren't here yet. 
Checked with the backend, they're of no use to them and won't be in any near future...
But we could, as a solution, write the tag headers in the middleware itself, after the response has been written.. so that we report what actually happened and what we know will happen, although could be done in another PR

Reply:
Ah of course, I was thinking that we weren't writing these to the _response_ as opposed to our spans! Seems fine to me as-is in that case, thanks 🙂